addquicktag.php

```diff
@@ -294,7 +294,9 @@ public function admin_enqueue_scripts() {
 		// Alternative to JSON function
 		// wp_localize_script( self :: get_textdomain() . '_script', 'addquicktag_tags', get_option( self :: $option_string ) );
 
-		if ( function_exists( 'is_gutenberg_page' ) && is_gutenberg_page() ) {
+        $current_screen = get_current_screen();
+        // In WP 5.0 those functions do not exist no more. Not sure, if they continue to exist in the plugin.
+		if ( isset($current_screen->is_block_editor) && $current_screen->is_block_editor ) {
```

Comment:
@websupporter Diese Methode ist bei mir im Test nie vorhanden, gibt es einen Trick? Umgebung WP 5.1-alpha-43678

Reply:
Soweit ich weiß hat 5.1, beziehungsweise der trunk von WordPress derzeit noch kein Gutenberg drin. Ich weiß nicht warum, aber die hatten 5.0 komplett in einem extra 5.0 Branch zusammengebaut und noch nicht in master gemergt.

Reply:
Danke dir, ich habe im Trunk nix, warum auch immer. Ich ziehe von http://core.svn.wordpress.org/trunk wo ich alles erwarten würde :(
Ich habe noch Problem, deine Ergänzung klappt gut; nun müssen aber noch die Klassen im `button` ergänzt, removed werden, hast du dazu Erfahrungen, wie Gutenberg das tut?

Reply:
Zum Kontext: https://make.wordpress.org/core/2018/12/11/wordpress-branch-status-update/ Wegen der Buttons guck ich die Tage mal.

Reply:
Das wäre sehr lecker. Danke.